src/Admin/Views/Users/Index.cshtml

```diff
@@ -72,8 +72,8 @@
                             }
                             @if (featureService.IsEnabled(Bit.Core.FeatureFlagKeys.MembersTwoFAQueryOptimization))
                             {
-                                var usersTwoFactorIsEnabled = TempData["UsersTwoFactorIsEnabled"] as IEnumerable<(Guid userId, bool twoFactorIsEnabled)>;
-                                @if(usersTwoFactorIsEnabled.FirstOrDefault(tuple => tuple.userId == user.Id).twoFactorIsEnabled)
+                                @if(TempData["UsersTwoFactorIsEnabled"] is IEnumerable<(Guid userId, bool twoFactorIsEnabled)> usersTwoFactorIsEnabled
+                                    && usersTwoFactorIsEnabled.FirstOrDefault(tuple => tuple.userId == user.Id).twoFactorIsEnabled)
```

Comment:
If we're using `FirstOrDefault` then we should handle the `null` case (I assume default to `false`). Here we _should_ have a 2FA result for each user in the result set, so I think just `First` would be fine. That said, I haven't tested that locally, and I would be OK if you wanted to take the more cautious approach.

Reply:
Added a couple checks. The Roslyn analyzer says that the default value of `ValueTuple<Guid, bool>` isn't null, so I broke up the `if` statement into separate variables and used pattern for null checking.